engine/include/engine/graphics/OpenGL.hpp

```diff
@@ -132,6 +133,27 @@ namespace engine::graphics {
         * @brief Clears GL_DEPTH_BUFFER_BIT, GL_COLOR_BUFFER_BIT, and GL_STENCIL_BUFFER_BIT.
         */
         static void clear_buffers();
+#
+
+
+        static unsigned int create_framebuffer();
+        static unsigned int create_color_attachment(int SCR_WIDTH, int SCR_HEIGHT);
+        static unsigned int create_render_buffer(int SCR_WIDTH, int SCR_HEIGHT);
+        static unsigned int configure_framebuffer_rectangle();
+        static void bind_framebuffer(unsigned int framebuffer);
+        static void bind_colorbuffer(unsigned int colorbuffer);
```

Comment:
Da li bind_colorbuffer praktično radi isto što i aktiviranje teksture? Ako da, možda bi bilo korisno razmotriti da li je potrebno zadržati oba naziva ili ih ujediniti radi jasnije organizacije koda.